MicrosoftFluentUI.podspec

```diff
@@ -97,6 +97,7 @@ Pod::Spec.new do |s|
 
   s.subspec 'CommandBar_ios' do |commandbar_ios|
     commandbar_ios.platform = :ios
+    commandbar_ios.dependency 'MicrosoftFluentUI/BottomSheet_ios'
```

Comment:
we should have a new subspec with muticommandbar instead? and it has commandbar dependencies?